extensions/QCOM/SPV_QCOM_image_processing.asciidoc

```diff
@@ -0,0 +1,285 @@
+SPV_QCOM_image_processing
+==========================
+
+Name Strings
+------------
+
+SPV_QCOM_image_processing
+
+Contact
+-------
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/KhronosGroup/SPIRV-Headers
+
+Contributors
+------------
+
+- Balaji Calidas, Qualcomm
+- Jeff Leger, Qualcomm
+- Ruihao Zhang, Qualcomm
+- Wooyoung Kim, Qualcomm
+
+Notice
+------
+
+Copyright (c) 2020 The Khronos Group Inc. Copyright terms at
+http://www.khronos.org/registry/speccopyright.html
+
+Status
+------
+
+- Approved by the SPIR-V Working Group: 2022-09-07
```

Comment:
I think this might be copied from another extension, what's the actual status?

Reply:
Fixed it to "Final"